### PR TITLE
Ignore .vs 

### LINKED ISSUE #83: Remove src/cartservice/.vs
https://github.com/GoogleCloudPlatform/microservices-demo/tree/master/src/cartservice/.vs/cartservice

This directory probably should not be checked in. Contains many binary/cache files. If there's config, we can keep it, but I couldn't find it as such.

### PR DESCRIPTION
**Pull request by orthros**
Fixes #83

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@ pkg/
 *.swp
 *~
 .vscode/
+.vs/
 .vs/slnx.sqlite
```
Comment (ahmetb) [inline .gitignore]: I think you can remove these 2 lines now?
also you might need to do `git rm -rf .vs/`

Comment (orthros) [reply .gitignore]: My previous commit should have removed the only reference .vs in the codebase `find . -name ".vs"` returns nothing

**Comment by ahmetb** [comment]
@simonz130 are these OK to delete do you think?

**Comment by simonz130** [comment]
NP to remove what you already merged.
Next time give me more time to respond.

**Comment by ahmetb** [comment]
Will do. I quickly double checked to see if .vs is ok to keep out of source control, seemed pretty straightforward.

**Pull request merged**